Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs

```diff
@@ -2032,6 +2032,78 @@ public void Devices_TouchesBecomeStationaryWhenNotMovedInFrame()
         Assert.That(device.allTouchControls[0].phase.ReadValue(), Is.EqualTo(PointerPhase.Stationary));
     }
 
+    [Test]
+    [Category("Devices")]
+    public void Devices_TouchesWithSameIdDontGetStuck()
```

Comment:
Please name TODO_Devices_TouchesWithSameIdDontGetStuck. Regular tests should be green.